src/Evolution/Systems/Cce/Components/CharacteristicEvolution.hpp

```diff
@@ -73,12 +93,86 @@ struct CharacteristicEvolution {
       tmpl::list<Actions::InitializeCharacteristicEvolutionVariables,
                  Actions::InitializeCharacteristicEvolutionTime,
                  Actions::InitializeCharacteristicEvolutionScri,
+                 Actions::RequestBoundaryData<
+                     typename Metavariables::cce_boundary_component,
+                     CharacteristicEvolution<Metavariables>>,
+                 Actions::ReceiveWorldtubeData<Metavariables>,
+                 ::Actions::MutateApply<
```

Comment:
we would prefer a new action or (incorporate into ReceiveWorldtubeData) that just combines all of these MutateApply's  into a single action.  Is there any reason to have them separate?

Reply:
No particular reason to split them. Would you be satisfied with a typelist? Given that the additional action would just be a sequence of `mutate_apply` calls it doesn't seem to add a whole lot aside from the puffy template code, documentation, and tests that would have to be included with a new action.